doc/api/http.md

```diff
@@ -1959,6 +1959,17 @@ added: v11.6.0
 Read-only property specifying the maximum allowed size of HTTP headers in bytes.
 Defaults to 8KB. Configurable using the [`--max-http-header-size`][] CLI option.
 
+## http.maxUriSize
+<!-- YAML
+added: v12.0.0
+-->
+
+* {number}
+
+Read-only property specifying the maximum allowed size of HTTP request uri
+in bytes. Defaults to 7KB. Configurable using the
+[`--max-http-uri-size`][] CLI option.
```

Comment:
Should we really allow modifying the maximum uri size? AFAIK we had some kind of internal limit as well so far. Why don't we just keep that as it is?

Reply:
I'm not aware of an internal limit to the URI size before this change. What we did have was the  maxHeaderSize, which kind acted like the limit because the http_parser or llhttp implementation also counted all the bytes before the headers and compared to it [[1]](https://github.com/nodejs/http-parser/blob/master/http_parser.c#L154)[[2]](https://github.com/nodejs/node/blob/master/src/node_http_parser_impl.h#L832), but then again it caused the ambiguity the PR is trying to fix. Is that what you're referring to?